common/test/acceptance/pages/lms/problem.py

```diff
@@ -435,6 +443,23 @@ def is_correct_choice_highlighted(self, correct_choices):
 
         return True
 
+    def is_submitted_choice_highlighted(self, correct_choices):
+        """
+        Check if submitted answer/choice highlighted for choice group.
+        """
+        submitted_status_xpath = '//fieldset/div[contains(@class, "field")][{0}]/label[contains(@class, "choicegroup_submitted")]/span[contains(@class, "status submitted")]'  # pylint: disable=line-too-long
```

Comment:
Given the complexity of this code, can you refactor so that `is_submitted_choice_highlighted` and `is_correct_choice_highlighted` share a helper method? If I'm reading this correctly, the only difference is the classname (`choicegroup_submitted`).